app/src/main/java/com/fstyle/structure_android/MainActivity.kt

```diff
@@ -0,0 +1,15 @@
+package com.fstyle.structure_android
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/04a1b709612ae5ecce8bed53309e05fa09d012b8/app/src/main/java/com/fstyle/structure_android/MainActivity.kt#L1"></span>Package name should match the pattern: ^[a-z]+(.[a-z][A-Za-z0-9]<em>)</em>$</td>
    </tr>
  </tbody>
</table>